openassessment/xblock/static/sass/oa/utilities/_shame.scss

```diff
@@ -4,6 +4,7 @@
 // NOTES: use for any bad-form/orphaned scss that knowingly violate edX FED architecture/standards (see - http://csswizardry.com/2013/04/shame-css)
 
 .openassessment {
+  @include direction();
```

Comment:
Also, is there a specific reason why this is in the _shame.scss Sass partial? Do you need at the end of the compilation? Or something else?